pkg/streamer/slicestreamer.go

```diff
@@ -0,0 +1,18 @@
+package streamer
+
+import "github.com/operator-framework/operator-registry/pkg/api"
+
+// NewSliceStreamer returns a streamer whose Send function appends the
+// bundle object to a slice of bundles.
+func NewSliceStreamer() *sliceStreamer {
+	return &sliceStreamer{}
+}
+
+type sliceStreamer struct {
+	Slice []*api.Bundle
+}
+
+func (s *sliceStreamer) Send(b *api.Bundle) error {
+	s.Slice = append(s.Slice, b)
+	return nil
+}
```

Comment:
Could this avoid the struct completely:

```suggestion
type sliceStreamer []*api.Bundle
```

And let users do things like:

```
DoThing(sliceStreamer(bundles))
```

without needing to have a `New...` function?

Reply:
Resolved in the latest commit.